UI/xdg-data/com.obsproject.Studio.Gnome.desktop

```diff
@@ -0,0 +1,87 @@
+Comment[el_GR]=Δωρεαν Λογισμικό Streaming/Kαταγραφή ανοιχτου κωδικα
+GenericName[en_GB]=Streaming/Recording Software
+Comment[en_GB]=Free and Open Source Streaming/Recording Software
+GenericName[es_ES]=Software de retransmisión/grabación
+Comment[es_ES]=Software de retransmisión/grabación de código abierto y gratuito
+GenericName[et_EE]=Video voogesituse ja salvestuse tarkvara
+Comment[et_EE]=Tasuta ja avatud lähtekoodiga video voogesituse ja salvestuse tarkvara
+GenericName[fa_IR]=نرم افزار جریان/ضبط
+Comment[fa_IR]=نرم افزار منبع باز و رایگان جریان/ضبط
+GenericName[fr_FR]=Logiciel de diffusion/enregistrement
+Comment[fr_FR]=Logiciel de diffusion/enregistrement gratuit et Open Source
+GenericName[he_IL]=תוכנה לשידורים חיים והקלטה
+Comment[he_IL]=תכנה חינמית בקוד פתוח לשידורים חיים ולהקלטה
+GenericName[hi_IN]=स्ट्रीमिंग/रिकॉर्डिंग सॉफ्टवेयर
+Comment[hi_IN]=स्वतंत्र एवं खुले स्रोत वाला स्ट्रीमिंग/रिकॉर्डिंग सॉफ्टवेयर
+GenericName[id_ID]=Perangkat Lunak Streaming/Perekaman
+Comment[id_ID]=Perangkat Lunak Streaming/Perekaman Gratis dan Sumber Terbuka
+GenericName[it_IT]=Streaming/Software Di Registrazione
+Comment[it_IT]=Software Libero e Open Source Streaming/Registrazione
+GenericName[ja_JP]=配信/録画ソフトウェア
+Comment[ja_JP]=無料のオープンソース配信/録画ソフトウェア
+GenericName[ka_GE]=ვიდეოს ეთერში გამშვები/ჩამწერი პროგრამა
+Comment[ka_GE]=თავისუფალი და ღია წყაროს მქონე, ვიდეოს ეთერში გამშვები/ჩამწერი პროგრამა
+GenericName[ko_KR]=방송 및 녹화 프로그램
+Comment[ko_KR]=무료 오픈소스 방송 및 녹화 프로그램
+GenericName[ms_MY]=Perisian Penstriman/Rakaman
+Comment[ms_MY]=Perisian Penstriman/Rakaman Bersumber Terbuka dan Bebas
+GenericName[nl_NL]=Streaming/Opname Software
+Comment[nl_NL]=Vrij en Open Source Streaming/Opname Sofware
+GenericName[pl_PL]=Oprogramowanie do transmisji strumieniowej/nagrywania
+Comment[pl_PL]=Darmowe i otwarte oprogramowanie do transmisji strumieniowej/nagrywania
+GenericName[pt_BR]=Software de Streaming/Gravação
+Comment[pt_BR]=Software de Streaming/Gravação de Código Aberto e Livre
+GenericName[pt_PT]=Programa de transmissão/gravação
+Comment[pt_PT]=Programa de transmissão/gravação livre e de código aberto
+GenericName[ro_RO]=Program de Streaming/Înregistrare
+Comment[ro_RO]=Program de streaming / înregistrare gratuit și open source
+GenericName[ru_RU]=Программа для видеостриминга и видеозаписи
+Comment[ru_RU]=Свободное и открытое ПО для видеостриминга и видеозаписи
+GenericName[sk_SK]=Streamovací/Nahrávací Software
+Comment[sk_SK]=Bezplatný a otvorený streamovací/nahrávací software
+GenericName[sv_SE]=Programvara för strömning/inspelning
+Comment[sv_SE]=Fri programvara för strömning/inspelning med öppen källkod
+GenericName[tr_TR]=Yayın/Kayıt Yazılımı
+Comment[tr_TR]=Ücretsiz ve Açık Kaynaklı Yayın/Kayıt Yazılımı
+GenericName[uk_UA]=Програма для трансляцій/запису
+Comment[uk_UA]=Вільне та відкрите програмне забезпечення для трансляцій/запису
+GenericName[zh_CN]=直播/录像软件
+Comment[zh_CN]=自由且开源的用于直播串流以及视频录制的软件
+GenericName[zh_TW]=串流與錄影軟體
+Comment[zh_TW]=免費，開源的串流與錄影軟體
```

Comment:
From my understanding these translations need to be moved above `[Desktop Action new-window]` into the section `[Desktop Entry]`.

(If this PR gets merged I will take care about the new strings ("New Wayland Instance" & "New Instance") and its translations for both files.)

Reply:
Fixed it even if my hope of getting this merged is fading

Reply:
These kinds of comments are not helpful. This PR was submitted after 27 RC1 - that is, within a feature freeze. We released 27.0.0 (a very large major version) 9 days ago. The past 9 days has been post-release collection of bug reports, verifying bug reports, fixing bugs, testing bugfixes, etc - this is also a period of feature freeze. Additionally, there are a number of other PRs to also be reviewed.

Reply:
It's not because this PR is "old", it's because the only argument given to me here were like from my pint of view: *it's a Qt bug, we shall wait for a fix rather than putting a workaround*